sample/applicationinsights-angularplugin-sample/package.json

```diff
@@ -31,7 +31,7 @@
     "@angular/platform-browser": "^14.2.0",
     "@angular/platform-browser-dynamic": "^14.2.0",
     "@angular/router": "^14.2.0",
-    "@microsoft/applicationinsights-angularplugin-js": "^3.0.3",
+    "@microsoft/applicationinsights-angularplugin-js": "../../dist/applicationinsights-angularplugin-js",
```

Comment:
This "should" be `^14.0.0` and yes it will only resolve after we publish that version number.

Please make sure to update this as part of the release PR (if not here)